Formula/wallpaper.rb

```diff
@@ -14,6 +14,7 @@ class Wallpaper < Formula
   end
 
   depends_on xcode: ["11.4", :build]
+  depends_on :macos
   depends_on macos: :sierra
```

Comment:
I feel like there's a lot of redundancy in these three lines.

Reply:
Almost; we had this situation elsewhere.

`depends_on macos: :sierra` is a no-op on linux. Same for the Xcode dep.

The goal is to have a clean error message on `brew install` on Linux:

```
wallpaper: macOS is required for this software.
Error: An unsatisfied requirement failed this build.
```

 I could remove the checks around high Sierra and the old Xcode version, as we don't care anymore. Is that OK?

Reply:
I think we still need the Xcode check, as it implicitly rules out Mojave (only has 11.3).

Reply:
Alternatively, we can make it
```ruby
depends_on macos: :catalina
```

Reply:
I went for the Xcode check.